tests/create_LWE.py

```diff
@@ -0,0 +1,242 @@
+from addict import Dict
+
+from CADETProcess.simulator import Cadet
+
+from CADETProcess.processModel import (
+
+    ComponentSystem,
+    FlowSheet,
+
+    Inlet,
+    Outlet,
+    Cstr,
+    GeneralRateModel,
+    TubularReactor,
+    LumpedRateModelWithoutPores,
+    LumpedRateModelWithPores,
+    MCT,
+
+    Linear,
+    Langmuir,
+    LangmuirLDF,
+    BiLangmuir,
+    BiLangmuirLDF,
+    FreundlichLDF,
+    StericMassAction,
+    AntiLangmuir,
+    Spreading,
+    SelfAssociation,
+    BiStericMassAction,
+    MultistateStericMassAction,
+    SimplifiedMultistateStericMassAction,
+    Saska,
+    GeneralizedIonExchange
+)
+
+
+def create_lwe(n_comp=4, unit_type=None, **kwargs) -> None:
+
+    def configure_all():
+        unit_types = [
+            'cstr', 'generalratemodel', 'tubularreactor',
+            'lumpedratemodelwithoutpores', 'lumpedratemodelwithpores', 'mct'
+        ]
+        for unit_type in unit_types:
+            configure_single_unit(unit_type)
+
+    def configure_single_unit(type_):
+        component_system = ComponentSystem(n_comp)
+        flow_sheet = FlowSheet(component_system)
+
+        if unit_type == 'cstr':
+            configure_cstr(component_system, kwargs)
+        elif unit_type == 'generalratemodel':
+            configure_grm(component_system, kwargs)
+        elif unit_type == 'tubularreactor':
+            configure_tb(component_system, kwargs)
+        elif unit_type == 'lumpedratemodelwithoutpores':
+            configure_lrm(component_system, kwargs)
+        elif unit_type == 'lumpedratemodelwithpores':
+            configure_lrmp(component_system, kwargs)
+        elif unit_type == 'mct':
+            configure_mct(component_system, kwargs)
+        else:
+            raise ValueError(f'Unknown unit operation type {type_}')
+
+    if unit_type is None:
+        configure_all()
```

Comment:
[question] I guess I don't fully understand this yet, what would `configure_all` do? Create a `Process` for each of the unit operation types?

Reply:
The idea was that if you don't specify a unit type it would create and run a simulation for all units.

Reply:
Decided to handle that a bit different anyways now :)

Reply:
> The idea was that if you don't specify a unit type it would create and run a simulation for all units.

I understand but I would suggest decoupling the configuration function from running the tests (where I would then iterate)

Reply:
> Decided to handle that a bit different anyways now :)

gotcha :nerd_face: